examples/core/picking/app.js

```diff
@@ -39,16 +43,15 @@ const animationLoop = new AnimationLoop({
       }
     })
     .then(textures => PLANETS.map(
-      (planet, i) => new Sphere({
-        gl,
+      (planet, i) => new Sphere(gl, {
```

Comment:
@ibgreen , this fixed Sphere geometry issue, before this change, sphere geometry options (nlat, nlong, etc) are ignored.